assets/js/admin/marketplace-suggestions.js

```diff
@@ -0,0 +1,340 @@
+/* global marketplace_suggestions ajaxurl */
```

Comment:
You may want to ask someone from Isotope to review this file since I'm more confortable with PHP than JS.

Reply:
@justinshreve would you be able to take a quick look over this for best practices? tia